.gitignore

```diff
@@ -10,6 +10,7 @@ npm-debug.*
 *.orig.*
 web-build/
 .env
+package-lock.json
```

Comment:
This should not be gitignored. See link:

https://stackoverflow.com/a/44210813

Reply:
please see my most recent commit